test/built-ins/String/fromCodePoint/argument-is-not-number.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
I think a better name for this file would be `argument-not-number-coercible.js` or simply `argument-not-coercible.js`  because (as your other tests confirm), `String.fromCodePoint` does work with non-number values.